.github/workflows/linkml-validation.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: LinkML Linting GitHub Action
+on:
+  push:
+    paths:
+      - "src/mixs/schema/mixs.yaml"
+jobs:
+  linkml:
+    permissions:
+      contents: write
+      pull-requests: write
+    name: LinkML Linting
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install linkml
+
+      - name: Run LinkML Linting
+        run: |
+          ## Results file header and basic formatting
+          printf '# LinkML Linting Results \n\n' > linting-results.md
+          printf "$(linkml --version) linting found the following issues. \n\n" >> linting-results.md
+          printf "_For more information about the linting rule categories, see the [LinkML linter documentation](https://linkml.io/linkml/schemas/linter.html#rules)._\n" >> linting-results.md
+          ## Lint the schema, and further format
+          linkml lint --config linkml-lint-config.yml --ignore-warnings -f markdown src/mixs/schema/mixs.yaml >> linting-results.md
```

Comment:
Aha! That works!

In regards to make, there are a variety of reasons why I've made it separate but to summarise the two main:

1. I have 0 experience with makefiles (which is main reason why I didn't consider)
2. I would argue such linting is a different priority from testing - the current `make test` suite is doing testing _and_ building I think? Which I personally would not do - I would rather go lint first (quick fixes) > run tests (maybe bigger fixes) > build; and run these either in parallel or link the first will fail the second and third if there is a failure first (As there is no point in burning through the latter); but if everything is OK might as well get them done as fast as possible (if that makes sense)

Reply:
(e.g, you would only get your linting results very late, only once everything is tested and built,m because we would have to put the upload action after the `make test` action step)